eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration/Parser/Common.php

```diff
@@ -168,6 +168,7 @@ public function mapConfig(array &$scopeSettings, $currentScope, ContextualizerIn
             $contextualizer->setContextualParameter('binary_dir', $currentScope, $scopeSettings['binary_dir']);
         }
         if (isset($scopeSettings['api_keys']['google_maps'])) {
+            $contextualizer->setContextualParameter('api_keys', $currentScope, ['google_maps' => $scopeSettings['api_keys']['google_maps']]);
             $contextualizer->setContextualParameter('api_keys.google_maps', $currentScope, $scopeSettings['api_keys']['google_maps']);
```

Comment:
@andrerom Why dont you add the whole `$scopeSettings['api_keys']` array as the value? It is not forward compatible this way.

One more suggestion would be replacing isset check by array_key_exists above. It is not possible to set null value as default (parameters.yml.dist eg) because if it stays null then neither of these contextual parameters will be defined and more things will fail. I can use false, but null would be more straightforward.